Makefile

```diff
@@ -88,10 +88,37 @@ install:
 docker:
 	DOCKER_BUILDKIT=1 docker build --pull --rm \
 		--build-arg LDFLAGS='$(LDFLAGS)' \
-		-t zqd:latest -t localhost:5000/zqd:latest -t localhost:5000/zqd:$(VERSION) .
+		-t zqd:latest \
+		-t localhost:5000/zqd:latest \
+		-t localhost:5000/zqd:$(VERSION) \
+		-t $(ZQD_ECR_HOST)/zqd:$(VERSION) \
+		.
+
+docker-push-local: docker
 	docker push localhost:5000/zqd:latest
 	docker push localhost:5000/zqd:$(VERSION)
 
+docker-push-ecr: docker
+	aws ecr get-login-password --region us-east-2 | docker login \
+	  --username AWS --password-stdin 792043464098.dkr.ecr.us-east-2.amazonaws.com/zqd
```

Comment:
How do we ensure different developers images don't collide when they build & push zqd to the same ECR host?

Reply:
The only way developers could collide with the current system is if their version tags, e.g.
"git describe --tags --dirty --always"
are identical. Now that I think about it, if two developers have identical branches, with no committed changes, then their version tags could be the same digest + dirty -- it would not distinguish between two non-identical dirty branches.
Is my logic correct here? If so, there is a possibility of collision and maybe we should disambiguate the tags by adding branch name.

Reply:
I disambiguated the zqd version by appending the K8s username.